### PR TITLE
Fixing problem with validation of array

### PR DESCRIPTION
**Pull request by ONyzhnyk-Luxoft**
Fixes #[[2451](https://github.com/SmartDeviceLink/sdl_core/issues/2451)]

This PR is **[ready]** for review.

### Risk
This PR makes **[no]** API changes.

### Summary
[buffer size for reading data from ini file was increased
tested with required parameters: Expected Behavior is in accordance with Observed ]

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: tatus [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+     Modified default URLs to make it clearer that they should be customized by the user
+
+[33mcommit dfbc2220c44ab5cd4a1fdf29dc115579809920a0[m
+Merge: 163affd 07c274f
+Author: JackLivio <jack@livio.io>
+Date:   Wed Apr 4 17:02:24 2018 -0400
+
+    Merge pull request #2110 from smartdevicelink/fix/add_license_to_mb_files
+    
+    Fix/add license to mb files
```
Comment (LuxoftAKutsan) [inline tatus]: @ONyzhnyk-Luxoft remove redundant files

**Comment by LuxoftAKutsan** [comment]
@ONyzhnyk-Luxoft @vkushnirenko-luxoft  please check UT build

**Comment by vkushnirenko-luxoft** [comment]
29 - config_profile_test (Failed)

File: tatus (hunk 1 above)
Comment (ONyzhnyk-Luxoft) [reply tatus]: fixed in [bbd5bef](https://github.com/smartdevicelink/sdl_core/pull/2669/commits/bbd5befcabccece0e86621ffcddbfa5f76eddad7)

**Pull request merged**